pages/common/docker-network.md

```diff
@@ -0,0 +1,32 @@
+# docker network
+
+> Create and manage docker network.
+> More information: <https://docs.docker.com/engine/reference/commandline/network/>.
+
+- List all available and configured networks on docker daemon:
+
+`docker network ls`
+
+- Create user defined network:
+
+`docker network create --driver {{driver_name}} {{network_name}}`
+
+- Display detailed information on one or more networks:
+
+`docker network inspect {{network_name}}`
+
+- Connect a container to a network:
+
+`docker network connect {{network_name}} {{container_name/id}}`
+
+- Disconnect a container from a network:
+
+`docker network disconnect {{network_name}} {{container_name/id}}`
+
+- Remove one or more networks in docker daemon:
```

Comment:
Should be disconnected from any container before `rm`?
```suggestion
- Remove a space-separated list of networks:
```

Reply:
Yes. Should be disconnected from container before `rm` but docker gives you this detail if you try to remove network which is connected to containers and does not allow to remove it.

@navarroaxel, How about mentioning ` Remove one or more space-separated networks in docker daemon`?

Reply:
Based on `docker` page, the `docker rm` example. Maybe you can move the `prune` example above this and just say:
```suggestion
- Remove a space-separated list of unused networks:
```